src/itype.h

```diff
@@ -441,7 +441,7 @@ struct islot_gunmod : common_ranged_data {
     int sight_dispersion = -1;
 
     /**
-     *  If set (non-zero) mod functions as sight when recoil above mod @ref sight_dispersion */
+     *  If set (non-negative) mod functions as sight when recoil above mod @ref sight_dispersion */
```

Comment:
> Wasn't it already set anyways before?

No. It was set to 0, which in current master branch means unset.

As for the comments, I'd word it roughly like:
`For sights (see @ref sight_dispersion), this value affects time cost of aiming. Lower is better. In case of multiple usable sights, the one with lowest aim cost is used.`
And for dispersion:
`If this value is set (non-negative), this gunmod functions as a sight. A sight is only usable to aim by a character whose current @ref recoil is at or below this value.`
Not sure how to `@ref` to other structures, but it should be referring to `Character::recoil` or `player::recoil`.